### ISSUE TITLE
css and js not being excluded by default

### ISSUE THREAD

**Issue by FiveOFive** [netlify/plugin-csp-nonce#55]
I've installed the plugin with default configurations:

```
[[plugins]]
  package = "@netlify/plugin-csp-nonce"
```

The edge function logs show unnecessary invocations for .css and .js files. But the documentation makes it sound like these should be excluded by default: `The glob expressions of path(s) that should not invoke the CSP nonce edge function. Must be an array of strings. This value gets spread with common non-html filetype extensions (*.css, *.js, *.svg, etc).` 

```
Oct 24, 10:36:18 AM: 01HDHC4E info   Unnecessary invocation for https://<site>/static/js/main.4e1796ab.js {Oct 24, 10:36:18 AM: 01HDHC4E info     method: "GET",Oct 24, 10:36:18 AM: 01HDHC4E info     "content-type": "application/javascript; charset=UTF-8"Oct 24, 10:36:18 AM: 01HDHC4E info   }
Oct 24, 10:36:18 AM: 01HDHC4E info   Unnecessary invocation for https://<site>/static/css/main.c06f8476.css { method: "GET", "content-type": "text/css; charset=UTF-8" }
Oct 24, 10:36:19 AM: 01HDHC4F info   Unnecessary invocation for https://<site>/static/js/main.4e1796ab.js {Oct 24, 10:36:19 AM: 01HDHC4F info     method: "GET",Oct 24, 10:36:19 AM: 01HDHC4F info     "content-type": "application/javascript; charset=UTF-8"Oct 24, 10:36:19 AM: 01HDHC4F info   }
Oct 24, 10:36:19 AM: 01HDHC4F info   Unnecessary invocation for https://<site>/static/css/main.c06f8476.css { method: "GET", "content-type": "text/css; charset=UTF-8" }
```

**Comment by jasonbarry** [comment]
Hmmm, we already exclude ~50 common extension types [here](https://github.com/netlify/plugin-csp-nonce/blob/main/src/__csp-nonce.ts#L181), including `.js` and `.css`. Do you have a public URL that I could inspect for further debugging?

**Comment by FiveOFive** [comment]
This was our dev environment. Is there a good way for me to privately share a link to you?

We tried using the UI plugin first, but had disabled it before getting those logs. Not sure if anything could be conflicting there.

We've resolved the issue for now with manual config.

```
[[plugins]]
  package = "@netlify/plugin-csp-nonce"
  [plugins.inputs]
    excludedPath = [
      "**/*.css",
      "**/*.js"
    ]
```

**Comment by jasonbarry** [comment]
What you encountered may have been fixed by https://github.com/netlify/plugin-csp-nonce/pull/56, could you try without the `excludedPath` in your netlify.toml in a Deploy Preview and try again @FiveOFive? be sure to try with version 1.2.5+. thanks!

**Comment by FiveOFive** [comment]
> What you encountered may have been fixed by #56, could you try without the `excludedPath` in your netlify.toml in a Deploy Preview and try again @FiveOFive? be sure to try with version 1.2.5+. thanks!

Seems to be fixed. At least I am not able to reproduce it. Thanks!
